app/views/PaymentProblemsView.scala.html

```diff
@@ -32,39 +32,39 @@ <h1 class="header-margin-top">@messages("payment.problems.title")</h1>
     <p>@messages("payment.problems.p1")</p>
     <p>@messages("payment.problems.p2")</p>
 
-    <h2 class="govuk-heading-l">@messages("payment.problems.h2")</h2>
+    <h2 class="govuk-heading-l">@messages("payment.problems.h2.1")</h2>
```

Comment:
Amend title:
@main_template(title = messages("global.nav.title"))
to use: payment.problems.title